gamefixes-steam/292410.py

```diff
@@ -0,0 +1,9 @@
+""" Street Racing Syndicate
+"""
+#pylint: disable=C0103
+
+from protonfixes import util
+
+def main():
+    util.protontricks('lavfilters') # fix videos
```

Comment:
I don't own this game to test, but if the video problems can be solved via `util.disable_protonaudioconverter()` that would be preferred as it would require no additional codecs to be installed into the prefix

Reply:
Videos look like this without `lavfilters` and `util.disable_protonaudioconverter()` doesn't do anything.
![Screenshot_from_2024-02-22_12-19-38](https://github.com/Open-Wine-Components/ULWGL-protonfixes/assets/10759579/8cd53135-8075-48b5-a424-509444f2923c)